src/renderer/qssWidget/css/qssWidget.css

```diff
@@ -317,3 +317,78 @@ body {
 .fl-qssMenuWidget .fl-qssMenuWidget-controls .fl-qssWidgetMenu-item.active {
     animation: menuItemActivation 0.25s 2;
 }
+
+.fl-qssToggleWidget-extendedTipContainer {
```

Comment:
this file is getting a quite big, we should probably get it split

Reply:
Maybe we can split this file into 4 files - one for the widget as a whole and one file for each widget type. However, as you are working on  GPII-3575, there will be conflicts if I make any changes now. So I suggest that we postpone this until after we merge everything.